idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -351,12 +351,12 @@ def _make_performance(self):
         # ======================================================================
         # Aliases for  Sets
         vap_comp = self.config.vapor_side.property_package.component_list
-        liq_comp = self.config.liquid_side.property_package.solvent_set | \
-            self.config.liquid_side.property_package.solute_set
-        liq_comp_diffus = \
-            self.config.liquid_side.property_package.component_list_diffus_apparent
+        liq_comp = self.config.liquid_side.property_package.component_list
+        equilibrium_comp = vap_comp & liq_comp
         solvent_comp_list = \
-            self.config.liquid_side.property_package.component_list_solvent_apparent
+            self.config.liquid_side.property_package.solvent_set
+        electrolyte_comp_diffus = self.config.liquid_side.property_package.component_list_diffus\
```

Comment:
We are going to need to change this - the diffusing species set is not standard so will not be present in most property packages. At first glance, it appears this is only used for the concentrations at the interface, and I don't think we actually need those right now (a quick look made it seem like they were only used to calculate the enhancement factor).

Moving forward, I think we will hopefully be able to replace this with the full true species set.